src/function.rs

```diff
@@ -91,52 +90,71 @@ impl<'cb> ReturnValue<'cb> {
 /// including the receiver, the number and values of arguments, and
 /// the holder of the function.
 #[repr(C)]
-pub struct FunctionCallbackInfo(Opaque);
-
-impl InIsolate for FunctionCallbackInfo {
-  fn isolate(&mut self) -> &mut Isolate {
-    self.get_isolate()
-  }
+pub struct FunctionCallbackInfo {
+  implicit_args: *mut Opaque,
+  values: *mut Value,
+  length: int,
```

Comment:
This specifically matches the C++ class definition? What if say, the order of these changes? Seems fragile. This ideally would be solved by code generation, but could also be solved by using an opaque and adding bindings to retrieve the  members. Both solutions sound annoying - so I would suggest to just add a comment warning about this.

Reply:
I considered that too, and I'll do as you suggest and add a warning.
That said, the layout of this struct is effectively part of the v8 ABI as it is defined in v8.h and there are plenty of inline functions that (also defined in v8.h) that access its members. That makes it more-or-less fixed.
I also believe that the tests would catch a change in layout immediately.